Dockerfile

```diff
@@ -1,31 +1,30 @@
 FROM ubuntu:latest
-LABEL version="1.0" maintainer="John Stucklen <stuckj@gmail.com>"
+LABEL version="1.1" maintainer="John Stucklen <stuckj@gmail.com>"
 
+ENV LC_ALL en_US.UTF-8
+ENV LANG en_US.UTF-8
+ENV LANGUAGE en_US:en
 ENV SUBSONIC_UID 1000
 ENV SUBSONIC_GID 1000
 
-RUN apt update \
-  && apt upgrade -y \
-  && apt install -y locales ffmpeg openjdk-8-jre-headless nano flac lame mikmod timidity wget \
-  && apt clean \
+RUN export DEBIAN_FRONTEND=noninteractive \
+  && apt-get update \
+  && apt-get upgrade -y \
+  && apt-get install -y locales ffmpeg openjdk-8-jre-headless nano flac lame mikmod timidity wget \
+  && apt-get clean \
   && rm -rf /var/lib/apt/lists/*
 
 RUN sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
-    locale-gen
-ENV LC_ALL en_US.UTF-8 
-ENV LANG en_US.UTF-8  
-ENV LANGUAGE en_US:en 
+    locale-gen && \
+    localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
 
-RUN mkdir -p /opt/subsonic \
-  && localedef -i en_US -c -f UTF-8 -A /usr/share/locale/locale.alias en_US.UTF-8
-
-RUN wget --no-check-certificate https://s3-eu-west-1.amazonaws.com/subsonic-public/download/subsonic-6.1.6-standalone.tar.gz \
-  && tar xvzf subsonic-6.1.6-standalone.tar.gz -C /opt/subsonic \
-  && rm -rf subsonic-6.1.6-standalone.tar.gz
+RUN mkdir -p /opt/subsonic && \
+    wget --no-check-certificate https://s3-eu-west-1.amazonaws.com/subsonic-public/download/subsonic-6.1.6-standalone.tar.gz && \
```

Comment:
Nitpick. I like operators at the beginning so it's easier to not miss them (always in the same column). Also so if you remove the last line you can't forget to remove the operator for the last command since it's on the same line.

Also, it will keep it consistent with the above apt command that still has them at the beginning.

Reply:
I couldn't agree more. 

But, I also think it should be consistent throughout the file, so I've adjusted it to the style that is used in the rest of the file.

But, as I said, I agree 100% with your point, I'm happy to change that for all lines then ...

Reply:
Done.